omnibus/lib/ostools.rb

```diff
@@ -46,6 +46,10 @@ def heroku_target?()
     return ENV['AGENT_FLAVOR'] == 'heroku'
 end
 
+def ua_target?()
```

Comment:
Should we rename it to `universal_agent_target?` I feel like `ua` is not very descriptive. Also, you don't parenthesis for methods without arguments

Reply:
We are going to remove `universal` agent nomenclature because it is inherently confusing. That said, a cryptic name will remain for the time being as the product requirements for the package remain. This will be revised in the future.